pkg/operator/injection/injection.go

```diff
@@ -16,12 +16,14 @@ package injection
 
 import (
 	"context"
+	"flag"
 	"fmt"
+	"os"
 	"time"
 
 	"github.com/samber/lo"
 	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+	kubeerrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
Why do we need an alias for this?

Reply:
A previous iteration imported `errors`, this can go away now for sure.